examples/09_ui/main.rs

```diff
@@ -0,0 +1,226 @@
+//! Displays a shaded sphere to the user.
+
+extern crate amethyst;
+extern crate genmesh;
+
+use amethyst::assets::Loader;
+use amethyst::core::cgmath::Vector3;
+use amethyst::core::cgmath::prelude::InnerSpace;
+use amethyst::core::transform::Transform;
+use amethyst::ecs::World;
+use amethyst::prelude::*;
+use amethyst::renderer::{AmbientColor, Camera, DisplayConfig, DrawShaded, Light, Mesh, Pipeline,
+                         PngFormat, PointLight, PosNormTex, RenderBundle, RenderSystem, Rgba,
+                         ScreenDimensions, Stage};
+use amethyst::ui::{DrawUi, TtfFormat, UiBundle, UiImage, UiText, UiTransform};
+use amethyst::winit::{Event, KeyboardInput, VirtualKeyCode, WindowEvent};
+use genmesh::{MapToVertices, Triangulate, Vertices};
+use genmesh::generators::SphereUV;
+
+const SPHERE_COLOUR: [f32; 4] = [0.0, 0.0, 1.0, 1.0]; // blue
+const AMBIENT_LIGHT_COLOUR: Rgba = Rgba(0.01, 0.01, 0.01, 1.0); // near-black
+const POINT_LIGHT_COLOUR: Rgba = Rgba(1.0, 1.0, 1.0, 1.0); // white
```

Comment:
Just a note, I think the `Rgba` structure is redundant.

Reply:
This was copy pasted from the sphere example, should I change it there too?

Reply:
No, it's not related to this PR.